packages/cli/src/daemon/enable/Enable.handler.ts

```diff
@@ -192,8 +192,14 @@ export default class EnableDaemonHandler implements ICommandHandler {
                 .pipe(new tar.Parse())
                 .on('entry', function(entry: any) {
                     if (entry.type == "File" && (entry.path as string).includes(fileToExtract)) {
+                        const sysInfo: ISystemInfo = ProcessUtils.getBasicSystemInfo();
+                        let fileCreateOpts: any = {};
+                        if (sysInfo.platform == "linux" || sysInfo.platform == "darwin") {
```

Comment:
Curious about this: 
`sysInfo.platform == "linux" || sysInfo.platform == "darwin"`
vs this:
`sysInfo.platform !== "win32"`

It probably doesn't matter, as I don't we think support of the other systems that may be returned by [process.platform](https://nodejs.org/api/process.html#processplatform): `aix`, `freebsd`, `openbsd`, `sunos`

Reply:
That might be shorter. I just suggested that line because Windows is working right now. It might be that we don't even need to do a check and can just set the permission bits to 0o750 for every OS.